package.json

```diff
@@ -75,15 +75,15 @@
     "ember-modal-dialog": "2.2.0",
     "ember-moment": "7.3.1",
     "ember-normalize": "1.0.0",
-    "ember-page-title": "^3.1.2",
+    "ember-page-title": "^3.2.0",
```

Comment:
- 3.1.2 has an issue with ember-cli
- 3.2.0 fixes it, but there's a regression to an old issue we already have. I submitted a PR to fix it.